### PR TITLE
Adds Bones and sinew to butchering results. Adds bone armor(craftable)

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
<s>Sinew is (unused) as of now,</s> this is mostly just some "framework" for more tribal weapons/armor/items. 

:cl:
rscadd: Goliaths now drop bones, and Watchers drop sinews.
rscadd: You can craft primitive gear from bone and sinew.
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
We rust now

**Comment by TechnoAlchemisto** [comment]
dont even imply gun crafting on this pr lest we summon him

**Comment by Cheridan** [comment]
>We rust now

we even have the gratuitous nudity and the griefing

File: code/game/objects/items/stacks/sheets/mineral.dm [hunk 1]
```diff
@@ -372,3 +373,20 @@ var/global/list/datum/stack_recipe/abductor_recipes = list ( \
 /obj/item/stack/sheet/mineral/abductor/New(var/loc, var/amount=null)
 	recipes = abductor_recipes
 	..()
+
+
+/*
+ * Bones
+ */
+/obj/item/stack/sheet/mineral/bone
```
Comment (KorPhaeron) [inline code/game/objects/items/stacks/sheets/mineral.dm]: What does the mineral path actually do

**Comment by KazeEspada** [comment]
Rare drop for a goliath gem?

**Comment by TechnoAlchemisto** [comment]
maybe later ;)

@KorPhaeron nothing at all

**Comment by KorPhaeron** [comment]
You should probably make it not a mineral that's kind of weird though maybe it doesn't matter either.

Also please change the recipe of the oar in necropolis_chests.dm to use bone instead of hide

**Comment by TechnoAlchemisto** [comment]
Your Wish Is My Command.

**Comment by TechnoAlchemisto** [comment]
Sinew now has a use, although it is purely cosmetic.

File: code/modules/clothing/under/ties.dm [hunk 2]
```diff
@@ -305,4 +305,18 @@
...
+
+/obj/item/clothing/tie/talisman
+	name = "bone talisman"
+	desc = "A hunter's talisman, some say the old gods smile on those who wear it."
+	icon = 'icons/obj/clothing/ties.dmi'
+	icon_state = "talisman"
+	item_state = ""
+	item_color = "talisman"
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS|HEAD
+	armor = list(melee = 5, bullet = 5, laser = 5, energy = 5, bomb = 50, bio = 65, rad = 5) //Faith is the best armor. //This won't actually work because of accessories kill me with a fucking knife jesus christ I hate code
```
Comment (PKPenguin321) [inline code/modules/clothing/under/ties.dm]: throwback to https://github.com/tgstation/-tg-station/pull/12741

**Comment by FantasticFwoosh** [comment]
Layer it onto a wall girder with a little bone like a reinforced wall for meat walls

**Comment by PKPenguin321** [comment]
i think you fucked up your merge conflict fixing, you got my krav PR in there
you probably unticked all the files when you were committing, you're supposed to leave those ticked

**Comment by TechnoAlchemisto** [comment]
fixed

**Comment by octareenroon91** [comment]
I'm not sure what balance issues might creep up, but shouldn't more mobs have bone or sinew?

**Comment by KorPhaeron** [comment]
Harvesting from those human remains decals when

**Comment by TechnoAlchemisto** [comment]
Goliath bones are xtra strong and watcher sinew is extra sinewy so they make good crafting materials

**Comment by FantasticFwoosh** [comment]
To be honest, if we ever do go with a skeletal furniture theme, it'd be really great for making atmospheric set-pieces like the skeleton minibar as a table (or keep the current as a sprited rack)

**Comment by KorPhaeron** [comment]
Changelog

File: code/modules/clothing/under/ties.dm (hunk 2 above)
Comment (KazeEspada) [inline code/modules/clothing/under/ties.dm]: Can this give me a unique skill like auto-guard?

**Comment by Razharas** [comment]
Tablecrafting in - i approve

**Pull request merged**